src/implementations/twig/components/social-media-follow/social-media-follow.story.js

```diff
@@ -27,26 +34,17 @@ const getArgTypes = () => ({
       category: 'Content',
     },
   },
-  toggle_other: {
-    name: 'Show the "more" link',
-    type: { name: 'boolean' },
-    description: 'toggle the visibility of the "other social networks" link',
-    table: {
-      category: 'Content',
-    },
-  },
 });
 
 const prepareData = (data, args) => {
-  const isItThere = data.links.slice(-1)[0].label === 'Other social networks';
+  correctPaths(data);
+  const clone = JSON.parse(JSON.stringify(data));
```

Comment:
maybe spread syntax..? ` const clone = { ...data };`

Reply:
I tried this already, and if I'm not wrong that won't work correctly. The spread syntax does not create a real copy, just a reference, so when deleting of editing the "cloned" data set, it would also be applied to the original one

Reply:
forget it, sorry.. like if i never experienced this.. :D spread on the object works as a clone, but not for nested objects and we have plenty in our data..